.gitignore

```diff
@@ -82,14 +87,14 @@ src/ontology/sources/*/*.obo
 src/ontology/mondo-with-equivalents.*
 src/ontology/mondo-edit.obo.tmp.obo
 src/ontology/reasoned-plus-equivalents.owl
-/mondo-with-equivalents.*
-src/ontology/sources/*
+src/ontology/mirror
+src/ontology/mirror/*
+src/ontology/tbn.obo
+src/ontology/tbn.owl
+src/ontology/imports/*_newterms.txt
+src/ontology/compare/
+
 src/patterns/imports/seed.txt
-*.norm
-src/dosdp-tools-*
-src/ontology/.cogs
-src/ontology/imports/omo_terms.txt
-.vscode
-src/ontology/mondo-ncit.owl
-src/ontology/tbd/*
-mondo-tags-reasoned.owl
+src/patterns/imports/seed_sorted.txt
+
+src/scripts/exclusion_reasons.csv
```

Comment:
FYI: Added to `.gitignore`. Taken from [Google Sheet](https://docs.google.com/spreadsheets/d/18V-_j4b22LzCytxFEM3pBEl6-_ODJY06gpUwH0F6JVY/edit#gid=1644570180).